### PR TITLE
Rds/kjw/rule 23 10

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section23/Rule23-10.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. baseline_system_type_compare()
+
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_11]`
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-11, continue to rule logic: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_system_types_dict.keys() for applicable_sys_type in APPLICABLE_SYS_TYPES): APPLICABLE`
```
Comment (mkarpman) [inline docs/section23/Rule23-10.md:27]: Please add "undetermined" outcome

File: docs/section23/Rule23-10.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+**Applicability Checks:**  
+- create a list of the target system types: `APPLICABLE_SYS_TYPES = [HVAC_SYS.SYS_11]`
+- Get B-RMR system types: `baseline_system_types_dict = get_baseline_system_types(B-RMR)`
+
+- loop through the applicable system types: `for target_sys_type in APPLICABLE_SYS_TYPES:`
+    - and loop through the baseline_system_types_dict to check the system types of the baseline systems: `for system_type in baseline_system_types_dict:`
+        - do baseline_system_type_compare to determine whether the baseline_system_type is the applicable_system_type: `if((baseline_system_type_compare(system_type, target_sys_type, false)):`
+            - the systems in the list: baseline_system_types_dict[system_type] are sys-11 - loop through the list of systems: `for hvac_system_id in baseline_system_types_dict[system_type]:`
+            - `CONTINUE TO RULE LOGIC`
+            - otherwise, rule not applicable: `else: RULE_NOT_APPLICABLE`
```
Comment (claperle) [inline docs/section23/Rule23-10.md:32]: @weilixu  Should this be in the Rule Assertion section? That is how I did it my RDSs, not sure if this is something that matters.

File: docs/section23/Rule23-10.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+
+# Airside System - Rule 23-10 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-10  
+**Rule Description:** System 11 Fan volume shall be reset from 100% airflow at 100% load to minimum airflow at 50% load.
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/section23/Rule23-10.md]: changed

**Comment by weilixu** [comment]
@KarenWGard I see 23-10 is in the repo. Is this an update? Is it ready for Merge?

**Pull request merged**